client/knowledge_base_search/src/containers/ViewAllContainer/ViewAllContainer.jsx

```diff
@@ -120,7 +121,7 @@ class ViewAllContainer extends Component {
                     onClick={() => { onQuestionClick(query.question); }}
                   >
                     { query.question }
-                    { query.is_training_query && (
+                    { query.is_training_query && selectedFeature === 'trained' && (
```

Comment:
probably better to use the object value from the `TrainingContainer` static `featureTypes`